cms/static/js/views/course_info_helper.js

```diff
@@ -2,6 +2,7 @@ define(['codemirror', 'js/utils/handle_iframe_binding', 'utility'],
     function(CodeMirror, IframeBinding) {
         var editWithCodeMirror = function(model, contentName, baseAssetUrl, textArea) {
             var content = rewriteStaticLinks(model.get(contentName), baseAssetUrl, '/static/');
+            content = rewriteCdnLinksToStatic(content); // eslint-disable-line no-undef
```

Comment:
why no-undef here?

Reply:
Quality tests were failing due to Eslint violations. In order to minimize these violations, I disabled using this.